modules/apps/change-tracking/change-tracking-web/src/main/resources/META-INF/resources/publications/js/components/ChangeTrackingIndicator.js

```diff
@@ -438,13 +439,16 @@ export default function ChangeTrackingIndicator({
 	const renderConflictIcon = () => {
 		if (Liferay.FeatureFlags['LPD-20556'] && !!dangerIcon) {
 			return (
-				<ClayDropDown
-					alignmentPosition={Align.BottomCenter}
-					menuElementAttrs={{style: {maxWidth: '303px'}}}
+				<ClayPopover
+					alightPosition="bottom"
+					onShowChange={setOpenPopover}
+					show={openPopover}
 					trigger={
 						<ClayButton
 							aria-label="conflict-button"
 							className="change-tracking-conflict-button"
+							onMouseOut={() => setOpenPopover(false)}
```

Comment:
@ctang3 does this mean the moment you move off the icon the popover disappears?

Reply:
Yeah, there's a very slight delay before the popover disappears after mousing out.